### ISSUE TITLE
ModelInput conflicts

### ISSUE THREAD

**Issue by webtop** [dotnet/machinelearning-samples#600]
When building the sample app I get:

> The type 'ModelInput' in 'E:\Projects\c_sharp\myMLApp\myMLAppML.Model\DataModels\ModelInput.cs' conflicts with the imported type 'ModelInput' in 'myMLAppML.Model, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. Using the type defined in 'E:\Projects\c_sharp\myMLApp\myMLAppML.Model\DataModels\ModelInput.cs'.

I simply followed [this tutorial](https://dotnet.microsoft.com/learn/ml-dotnet/get-started-tutorial/intro)

I'm using VS2019 CE

Thanks.

**Comment by webtop** [comment]
Excuse my stupidity, I'm new to .NET!
I just noticed that was only a warning and that the issue for not building is:

> 2>------ Build started: Project: myMLApp, Configuration: Debug Any CPU ------
2>myMLAppML.Model\obj\Debug\netcoreapp2.1\myMLAppML.Model.AssemblyInfo.cs(14,12,14,54): error CS0579: Duplicate 'System.Reflection.AssemblyCompanyAttribute' attribute
2>myMLAppML.Model\obj\Debug\netcoreapp2.1\myMLAppML.Model.AssemblyInfo.cs(15,12,15,60): error CS0579: Duplicate 'System.Reflection.AssemblyConfigurationAttribute' attribute
2>myMLAppML.Model\obj\Debug\netcoreapp2.1\myMLAppML.Model.AssemblyInfo.cs(16,12,16,58): error CS0579: Duplicate 'System.Reflection.AssemblyFileVersionAttribute' attribute
2>myMLAppML.Model\obj\Debug\netcoreapp2.1\myMLAppML.Model.AssemblyInfo.cs(17,12,17,67): error CS0579: Duplicate 'System.Reflection.AssemblyInformationalVersionAttribute' attribute
2>myMLAppML.Model\obj\Debug\netcoreapp2.1\myMLAppML.Model.AssemblyInfo.cs(18,12,18,54): error CS0579: Duplicate 'System.Reflection.AssemblyProductAttribute' attribute
2>myMLAppML.Model\obj\Debug\netcoreapp2.1\myMLAppML.Model.AssemblyInfo.cs(19,12,19,52): error CS0579: Duplicate 'System.Reflection.AssemblyTitleAttribute' attribute
2>myMLAppML.Model\obj\Debug\netcoreapp2.1\myMLAppML.Model.AssemblyInfo.cs(20,12,20,54): error CS0579: Duplicate 'System.Reflection.AssemblyVersionAttribute' attribute
2>obj\Debug\netcoreapp2.2\myMLApp.AssemblyInfo.cs(14,12,14,54): error CS0579: Duplicate 'System.Reflection.AssemblyCompanyAttribute' attribute
2>obj\Debug\netcoreapp2.2\myMLApp.AssemblyInfo.cs(15,12,15,60): error CS0579: Duplicate 'System.Reflection.AssemblyConfigurationAttribute' attribute
2>obj\Debug\netcoreapp2.2\myMLApp.AssemblyInfo.cs(16,12,16,58): error CS0579: Duplicate 'System.Reflection.AssemblyFileVersionAttribute' attribute
2>obj\Debug\netcoreapp2.2\myMLApp.AssemblyInfo.cs(17,12,17,67): error CS0579: Duplicate 'System.Reflection.AssemblyInformationalVersionAttribute' attribute
2>obj\Debug\netcoreapp2.2\myMLApp.AssemblyInfo.cs(18,12,18,54): error CS0579: Duplicate 'System.Reflection.AssemblyProductAttribute' attribute
2>obj\Debug\netcoreapp2.2\myMLApp.AssemblyInfo.cs(19,12,19,52): error CS0579: Duplicate 'System.Reflection.AssemblyTitleAttribute' attribute
2>obj\Debug\netcoreapp2.2\myMLApp.AssemblyInfo.cs(20,12,20,54): error CS0579: Duplicate 'System.Reflection.AssemblyVersionAttribute' attribute
2>myMLAppML.ConsoleApp\Program.cs(48,24,48,34): warning CS0436: The type 'ModelInput' in 'E:\Projects\c_sharp\myMLApp\myMLAppML.Model\DataModels\ModelInput.cs' conflicts with the imported type 'ModelInput' in 'myMLAppML.Model, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. Using the type defined in 'E:\Projects\c_sharp\myMLApp\myMLAppML.Model\DataModels\ModelInput.cs'.
2>Done building project "myMLApp.csproj" -- FAILED.

Is any of this related directly to ML.NET, or do I have a more base issue?

Thanks.

**Comment by bamurtaugh** [comment]
Hi @webtop - No worries, welcome to .NET and ML.NET! When you say "building the sample app," which sample app are you building? Is it an app from this machinelearning-samples repository? Or is it from the tutorial you listed (if so, which step of the tutorial)? 

It appears your issue is with several duplicate attributes in your AssemblyInfo.cs file- this might be caused by several different issues (see https://github.com/dotnet/cli/issues/4710). Some developers have resolved this type of issue by commenting out duplicate attributes in AssemblyInfo.cs. Others  delete the bin directory, obj directory, and/or AssemblyInfo.cs file in the Properties directory, and then run `dotnet restore` and `dotnet build`.

**Comment by webtop** [comment]
Hi @bamurtaugh,

The sample app I meant was the tutorial. I completed the tutorial and built the solution which is when I got the errors.

After I commented out the assembly file comments, I got a different error stating that the solution had more than one entry point. I have selected .NET core 2.2 for the solution, but saw that it was targeting 2.1 in the myMLApp.csproj file. This lead me to take a look at the main project properties, and I noticed there was no Startup Object selected. I changed that to be myMLApp.Program and all was well with the world.

Trial and error :)
